tests/main/install-refresh-private/successful_login.exp

```diff
@@ -0,0 +1,12 @@
+spawn snap login $env(SPREAD_STORE_USER)
```

Comment:
hmm, this'll be the fourth copy of this thing. maybe we need to move it to lib...

Reply:
they were not quite identical though (strangely) but they could, I made them identical now, I think I would prefer to share them in a different PR thought